api/routers/legacy.py

```diff
@@ -176,7 +176,7 @@ async def sql_get_contributions(contributors: List):
         JOIN Players as pl on (pl.id = rs.reportingID)
         join Players as ban on (ban.id = rs.reportedID)
         WHERE 1=1
-            AND pl.name in :contributors
+            AND pl.normalized_name in :contributors
```

Comment:
Make sure to add a to_jagex_name normalization in the beginning of this function, to make sure that the name is cleaned prior to searching. Else if dirty data enters in through this route, it would not be cleaned properly to execute the necessary sql. The names should also be checked for valid Jagex names either prior, or after, cleaning. This could be done in a single function for conversion of list --> jagex names.